collect_app/src/main/java/org/odk/collect/android/gdrive/InstanceGoogleSheetsUploaderTask.java

```diff
@@ -80,7 +78,7 @@ protected Outcome doInBackground(Long... instanceIdsToUpload) {
                         uploader.uploadOneSubmission(instance, destinationUrl);
                         outcome.messagesByInstanceId.put(instance.getDbId().toString(), DEFAULT_SUCCESSFUL_TEXT);
 
-                        analytics.logEvent(SUBMISSION, "HTTP-Sheets", Collect.getFormIdentifierHash(instance.getFormId(), instance.getFormVersion()));
+                        Analytics.log(SUBMISSION, "HTTP-Sheets", Collect.getFormIdentifierHash(instance.getFormId(), instance.getFormVersion()));
```

Comment:
We can probably just get rid of this event now seeing as we're going to kill Google Drive support